runners/katacoda/index.ts

```diff
@@ -356,6 +356,35 @@ export class Katacoda extends Runner {
         return null;
     }
 
+    runExecuteCommand(runCommand: RunCommand) : RunResult {
+        let terminal = (runCommand.command.parameters.length > 1 && runCommand.command.parameters[1].asynchronous) 
+            ? this.getTerminal('executeCommand') 
+            : undefined;
+        
+        let command = runCommand.command.parameters[0];
```

Comment:
The variable 'command' is only used in the assignment of bashCommand. So can direcly use it there